packages/proteus/bower.json

```diff
@@ -0,0 +1,21 @@
+{
+  "dependencies": {
+    "libsodium.js": "0.5.4",
+    "wire-webapp-cbor": "2.1.2"
+  },
+  "ignore": [
```

Comment:
Ignore pattern can be cleaned up. We have default ignores here: https://github.com/wireapp/wire-web-packages/blob/master/.gitignore

Reply:
We can actually delete the whole file, right? Since we won't publish it on Bower.